pkg/webhook/mutatingwebhook.go

```diff
@@ -174,3 +191,53 @@ func parseSidecarContainerImage(pod *corev1.Pod) (string, error) {
 
 	return image, nil
 }
+
+func (si *SidecarInjector) supportsNativeSidecar() (bool, error) {
+	list, err := si.NodeLister.List(labels.Everything())
+	if err != nil {
+		return false, fmt.Errorf("failed to get cluster nodes: %w", err)
+	}
+
+	supportsNativeSidecar := true
+	for _, node := range list {
+		version, err := getRelease(node.Status.NodeInfo.KubeletVersion)
+		if version < "1.29" || err != nil {
+			if err != nil {
+				klog.Errorf(`invalid node gke version: could not get node "%s" k8s release from version "%s": "%v"`, node.Name, version, err)
```

Comment:
### Logging vs returning error
Same thought here, we could return an error here. However, this is less severe so we can just disable native sidecar support in this case.